job-descriptions/product-manager.md

```diff
@@ -0,0 +1,75 @@
+
+If you are passionate about making the world better through software, come join us!
+
+## About the role
+
+You'll work with engineering, sales, marketing, and design to plan, ship, and grow usage and engagement in specific areas of our product. As an early member of our product team, you'll have significant product ownership and will communicate directly with our users to drive continuous improvement.
+
+Developers [love our product](https://engineeringblog.yelp.com/2019/11/winning-the-hackathon-with-sourcegraph.html) and rely on it daily to ship and deploy better software. Our passionate users provide a constant stream of thoughtful feedback and love engaging with us to help them and their teams solve their toughest problems.
+
+Our [public roadmap](https://docs.google.com/document/d/1cBsE9801DcBF9chZyMnxRdolqM_1c2pPyGQz15QAvYI/edit) shows the product priorities for each engineering team and we use [OKRs](https://about.sourcegraph.com/company/okrs) to communicate company goals for each quarter.
+
+### Responsibilities
+
+You will:
+
+- Own an area of the product, and drive and inform its product roadmap.
+- Collaborate with engineers and designers to brainstorm, plan, and build the solutions to customer problems.
+- Gather customer feedback to validate priorities, improve documentation, and define product requirements.
+- Be transparent in your internal and external communication. Sourcegraph is open source, so most discussions are public or in channels where you communicate directly with our customers.
+- Be a passionate advocate for a delightful user experience.
+
+### Qualifications
+
+This is a very technical role and requires software engineering experience to effectively communicate with engineers and make educated decisions affecting product scope and performance at scale.
+
+As a result, we expect:
+
+- Experience working as a software engineer.
+- Strong understanding of Git and Git workflows.
+- Excellent written and verbal communication skills, technical writing ability, and use of data to back up your arguments.
+- Ability to stay focused on goals and strategically prioritize work.
+- Deep passion for developer tools and productivity enhancements.
+- Demonstrated leadership skills and ability to drive and influence product strategy across engineering, design, and distribution.
+
+### Nice-to-haves
+
+- Previous product management experience on a software development or developer tools product.
+- Experience building and/or contributing to developer tools.
+- Experience working remotely with globally distributed teams.
+
+## Location
+
+We are a remote-first, globally distributed team focused on results and employee happiness, so wherever you choose to work is up to you. We don't require you to be in any particular timezone or location.
```

Comment:
New wording LGTM for all job descriptions. Let's link to the new team page so people can see a concrete list of teammate locations.

https://about.sourcegraph.com/company/team